### DISCUSSION TITLE
filename attribute does not play well with code-fold

### DISCUSSION THREAD

**Discussion by royfrancis** [quarto-dev/quarto-cli#7831] [Q&A] [answered]
### Description

I am not sure if this is how it's supposed to work, but it just looks weird that the filename is floating alone when the code is folded.

![Screenshot 2023-12-07 at 15 02 35](https://github.com/quarto-dev/quarto-cli/assets/5083500/f839593d-3ef8-4b2f-a5d0-f87db0e9c588)

````
---
title: Filename position
format:
  html:
    code-fold: true
---

```{r}
date()
```

```{r}
#| filename: R
date()
```

```{ojs}
//| filename: OJS
Date()
```
````

`quarto 1.4.506`

**Comment by cscheid** [comment, marked as answer]
We're tracking this here, will be fixed in a few days https://github.com/quarto-dev/quarto-cli/issues/2683
